apps/divorce/div-dn/div-dn.yaml

```diff
@@ -9,7 +9,7 @@ spec:
     nodejs:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/div/dn:prod-caffc83-20230117092259 #{"$imagepolicy": "flux-system:div-dn"}
+      image: hmctspublic.azurecr.io/div/dn:pr-670-6bb2bc6-20230118183412 #{"$imagepolicy": "flux-system:demo-div-dn"}
```

Comment:
I wouldnt change this, rather the demo.yaml for dn

Reply:
Done